metadata-ingestion/docs/sources/iceberg/iceberg.md

```diff
@@ -0,0 +1,23 @@
+### Concept Mapping
+
+<!-- This should be a manual mapping of concepts from the source to the DataHub Metadata Model -->
+<!-- Authors should provide as much context as possible about how this mapping was generated, including assumptions made, known shortcuts, & any other caveats -->
+
+This ingestion source maps the following Source System Concepts to DataHub Concepts:
+
+<!-- Remove all unnecessary/irrevant DataHub Concepts -->
+
+| Source Concept | DataHub Concept | Notes |
+| -- | -- | -- |
+| `iceberg` | [Data Platform](docs/generated/metamodel/entities/dataPlatform.md) | |
+| Table | [Dataset](docs/generated/metamodel/entities/dataset.md) | Each Iceberg table maps to a Dataset named using the parent folders.  If a table is stored under `my/namespace/table`, the dataset name will be `my.namespace.table`.  If a [Platform Instance](https://datahubproject.io/docs/platform-instances/) is configured, it will be used as a prefix: `<platform_instance>.my.namespace.table`. |
+| [Table property](https://iceberg.apache.org/docs/latest/configuration/#table-properties) | [User (a.k.a CorpUser)](docs/generated/metamodel/entities/corpuser.md) | A configurable table property can be used as a source of CorpUser owner. |
+| [Table property](https://iceberg.apache.org/docs/latest/configuration/#table-properties) | CorpGroup | A configurable table property can be used as a source of CorpGroup owner. |
+| Parent folders| Container | Coming soon | 
```

Comment:
FYI: In Iceberg, I believe we tend to call this `warehouse` (or sometimes warehouse location).

It's most user-facing as the catalog property `warehouse`.

https://iceberg.apache.org/docs/latest/configuration/#catalog-properties

Reply:
Thanks for pointing this out, I forgot about the warehouse property.  In this case though, I'm referring to the intermediate folders between the `warehouse` and the metadata/data folders.  

For example, let's say I have a table at this ADLS location: `abfss://my_container@my_datalake.dfs.core.windows.net/iceberg/my/namespace/table`
with a `warehouse` property of `abfss://warehouse@hogwartsdatalakeu.dfs.core.windows.net/iceberg/`.  Then the containers would be `my` and `namespace` and the table name would be `table`.  You would then see this in DataHub:
- my
  - namespace
    - table

I will update a piece of code to refer to this warehouse property.